jdi-light-bootstrap-tests/src/test/resources/general.xml

```diff
@@ -3,11 +3,15 @@
 <suite name="Test suite" parallel="methods" thread-count="1">
     <test name="Tests">
         <packages>
-            <package name="io.github.epam.html.tests.elements.common"/>
+            <!--<package name="io.github.epam.html.tests.elements.common"/>
             <package name="io.github.epam.html.tests.elements.complex"/>
             <package name="io.github.epam.html.tests.elements.composite.*"/>
             <package name="io.github.epam.html.tests.site"/>
-            <package name="io.github.epam.html.tests.suspend"/>
+            <package name="io.github.epam.html.tests.suspend"/>-->
+
+            <package name="io.github.epam.bootstrap.tests.common"/>
```

Comment:
may be  `<package name="io.github.epam.bootstrap.tests.*"/>` is more suitable for us here, isn't it?

Reply:
ok